lib/sandbox/chai-postman.js

```diff
@@ -356,6 +357,20 @@ module.exports = function (chai) {
         this._obj = total; // @todo: do this the right way adhering to chai standards
     });
 
+    // @note directly tv4 assertions since chai-json-schema does not play well inside VM.
+    Assertion.addMethod('bodyWithJSONSchema', function (schema) {
+        new Assertion(this._obj).to.be.postmanResponse;
+        new Assertion(schema).to.be.an('object');
+        new Assertion(this._obj).to.have.jsonBody();
+
+        var result = tv4.freshApi().validateResult(this._obj.json(), schema);
```

Comment:
`tv4` will pass validations if schema is empty, so we need to also assert that `result.missing` is empty or falsey.